controller/proxy-injector/webhook_config.go

```diff
@@ -23,11 +23,10 @@ type WebhookConfig struct {
 	trustAnchor         []byte
 	configTemplate      *template.Template
 	k8sAPI              kubernetes.Interface
-	noInitContainer     bool
 }
 
 // NewWebhookConfig returns a new instance of initiator.
-func NewWebhookConfig(client kubernetes.Interface, controllerNamespace, webhookServiceName string, noInitContainer bool, rootCA *tls.CA) (*WebhookConfig, error) {
+func NewWebhookConfig(client kubernetes.Interface, controllerNamespace, webhookServiceName string, rootCA *tls.CA) (*WebhookConfig, error) {
```

Comment:
The webhook config template does not use the value of `noInitContainer` when rendering the template, so it doesn't need to be passed into this func or set on the `WebhookConfig` struct.